core/src/network/handler/store.rs

```diff
@@ -40,7 +41,23 @@ pub fn handle_store(dht_data: StoreEntryAspectData, context: Arc<Context>) {
                     .expect("Could not spawn thread for storing EntryAspect::Content");
             }
             EntryAspect::Header(header) => {
-                panic!(format!("unimplemented store aspect Header: {:?}", header));
+                context
+                    .log("debug/net/handle: handle_store: Got EntryAspect::Header. processing...");
+                thread::Builder::new()
+                    .name(format!(
+                        "store_header_content/{}",
+                        ProcessUniqueId::new().to_string()
+                    ))
+                    .spawn(move || {
+                        match context
+                            .block_on(hold_header_workflow(&header, context.clone()))
+                        {
+                            Err(error) => context.log(format!("err/net/dht: {}", error)),
+                            _ => (),
+                        }
+                    })
+                    .expect("Could not spawn thread for storing EntryAspect::Header");
```

Comment:
This does not seem right to me. Publishing headers so you can get them by their address means they are entries themselves, not an aspect of the entry the header is about, which is what this `EntryAspect::Header` means..